### PR TITLE
Weekly-News-29-April-2016

### PR DESCRIPTION
**Pull request by kimtryce**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-a78a809723/). :bow:

File: _posts/2016-04-29-weekly-news-openstack-summit.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  - Rackspace
+  - News
+authorIsRacker: true
+---
+
+{{ page.excerpt }}
+
+### Containers meetup at OpenStack Summit Austin
+
+It was great seeing so many faces at the containers meetup this week at the Openstack Summit in Austin, Texas. We had a full day of containers talks which we have posted below.
```
Comment (everett-toews) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: You might want to lead with the [recap](https://www.youtube.com/watch?v=AB1K49AkhDE) here and point out the fact that's it's only a 2m30s recap for those who don't have the time to dive into the hours of video below.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-ee7f720466/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-b988a25fa9/)

File: _posts/2016-04-29-weekly-news-openstack-summit.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Weekly news: Rackspace Cantina & OpenStack Summit Wrap Up"
+date: 2016-04-29
+comments: true
+author: Kim Tryce <kim.tryce@rackspace.com>
+published: true
+excerpt: >
+See the highlights from the Openstack Summit and the Carina Cantina in this weekly's wrap-up including recorded talks from our presenters Ross Kukulinski, Google's Tim Hockin, Lee Calcote, and our own Everett Toews.
```
Comment (everett-toews) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: Excerpt needs to be indented like [this](https://github.com/getcarina/getcarina.com/blame/master/_posts/2016-04-22-weekly-news-openstack-summit.md#L7-L8).

File: _posts/2016-04-29-weekly-news-openstack-summit.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Weekly news: Rackspace Cantina & OpenStack Summit Wrap Up"
```
Comment (kmsholcomb) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: wrap-up

File: _posts/2016-04-29-weekly-news-openstack-summit.md (hunk 2 above)
Comment (kmsholcomb) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: ... from the OpenStack Summit ...
... week's wrap-up, including ...

File: _posts/2016-04-29-weekly-news-openstack-summit.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  - Rackspace
+  - News
+authorIsRacker: true
+---
+
+{{ page.excerpt }}
+
+### Containers meetup at OpenStack Summit Austin
+
+It was great seeing so many faces at the containers meetup this week at the Openstack Summit in Austin, Texas. We had a full day of containers talks which we have posted below.  For a preview, we've included a spiffy recap video that is only 2:30 in length.  Enjoy!
```
Comment (kmsholcomb) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: OpenStack

File: _posts/2016-04-29-weekly-news-openstack-summit.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/AB1K49AkhDE" frameborder="0" allowfullscreen></iframe>
+
+View the [recap on YouTube](https://www.youtube.com/watch?v=AB1K49AkhDE).
+
+### Carina in Production
+
+[Ross Kukulinski] (https://twitter.com/rosskukulinski) of NodeSource and O'Reilly author provides insight and a use case of running Carina in production. "Without a doubt, Carina is the most forward thinking platform Rackspace has ever created." - Ross
```
Comment (kmsholcomb) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: [Ross Kukulinski] (https://twitter.com/rosskukulinski) of NodeSource, and an O'Reilly author, provides insight about and ...

File: _posts/2016-04-29-weekly-news-openstack-summit.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/xOEBMq6mf9I" frameborder="0" allowfullscreen></iframe>
+
+View [Ross' talk on YouTube](https://www.youtube.com/watch?v=xOEBMq6mf9I).
```
Comment (kmsholcomb) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: Ross's

File: _posts/2016-04-29-weekly-news-openstack-summit.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+### Characterizing and Contrasting Container Orchestrators
+
+Engineering guru [Lee Calcote](https://twitter.com/lcalcote) provides an overview of different contianer providers and the pros/cons of each.
```
Comment (kmsholcomb) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: ... different container providers and the pros and cons of each.

(watch the typo of "containers")

File: _posts/2016-04-29-weekly-news-openstack-summit.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+Google's [Tim Hockin](https://twitter.com/thockin) from the Kubernetes team blew away the audience with his assessment of the current state of containers and what they are up to at Google.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/f4xC48bP4h4" frameborder="0" allowfullscreen></iframe>
+
+View [Tim's talk on YouTube](https://www.youtube.com/watch?v=f4xC48bP4h4).
+
+### Effective Docker Swarm
+
+Last but certainly not least, our own [Everett Toews](https://twitter.com/everett_toews) provides a walk through on how to use Docker Swarm.
```
Comment (kmsholcomb) [inline _posts/2016-04-29-weekly-news-openstack-summit.md]: walkthrough

**Comment by kmsholcomb** [comment]
Hi @kimtryce: Just a few comments, inline. Thanks!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-371070c2b4/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-f32c7f3667/)

**Pull request merged**